src/module-5/pagination/index.html

```diff
@@ -6,8 +6,7 @@
   <title>Movies App | Pagination</title>
 
   <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Source+Sans+Pro:300,400,400i,700&display=fallback">
-  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.6.0/dist/css/bootstrap.min.css">
-  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.0.13/css/all.css">
+  <link rel="stylesheet" href="./style.css">
```

Comment:
slider and pagination are not working well when started from their `index.html`;  
slider is not fully visible on the page;  
pagination hits an Error if user misclicks the page
<img width="555" alt="Screenshot 2021-10-21 at 22 49 00" src="https://user-images.githubusercontent.com/90797940/138346957-da72e46d-d234-4999-b6f4-e59300168d63.png">